models/release.go

```diff
@@ -397,11 +404,67 @@ func DeleteReleaseByID(id int64, u *User, delTag bool) error {
 		if err != nil && !strings.Contains(stderr, "not found") {
 			return fmt.Errorf("git tag -d: %v - %s", err, stderr)
 		}
-	}
 
-	if _, err = x.Id(rel.ID).Delete(new(Release)); err != nil {
-		return fmt.Errorf("Delete: %v", err)
+		if _, err = x.Id(rel.ID).Delete(new(Release)); err != nil {
```

Comment:
update after delete?

Reply:
forgot to remove update, good catch, thx

Reply:
actually it is just in diff looks like this, if you look at the full file code there is no update before delete, update is from other function :)